src/nvim/ex_eval.c

```diff
@@ -1834,21 +1782,19 @@ void enter_cleanup(cleanup_T *csp)
   }
 }
 
-/*
- * See comment above enter_cleanup() for how this function is used.
- *
- * This function is a bit like ex_endtry() except that there was not actually
- * an extra try block around the part that failed and an error or interrupt
- * had not (yet) been converted to an exception when the cleanup autocommand
- * sequence was invoked.
- *
- * This function has to be called with the address of the cleanup_T structure
- * filled by enter_cleanup() as an argument; it restores the error/interrupt/
- * exception state saved by that function - except there was an aborting
- * error, an interrupt or an uncaught exception during execution of the
- * cleanup autocommands.  In the latter case, the saved error/interrupt/
- * exception state is discarded.
- */
+/// See comment above enter_cleanup() for how this function is used.
```

Comment:
put this below first paragraph (as first line/paragraph is expected to state what the function does)